src/ast/field_analyser.cpp

```diff
@@ -294,7 +294,11 @@ int FieldAnalyser::analyse()
   std::string errors = err_.str();
   if (!errors.empty())
   {
+#ifdef FUZZ
+    (void)out_;
```

Comment:
Both the field analyser and semantic allow setting `out` in the constructor. Setting it to `/dev/null` from fuzz_main.cpp seems cleaner than doing ifdefs everywhere

Reply:
I didn't notice it. Thanks, fixed it.